frontend/src/fixtures/rideRequestFixtures.js

```diff
@@ -0,0 +1,52 @@
+const rideRequestFixtures = {
+    oneRideRequest: {
+        "userid": 1,
+        "day": "Monday",
+        "fullName": "Bob John",
```

Comment:
Has "fullName" but should reflect changes to backend table

Reply:
Agreed with Andrew: the fullName should come through userid instead.

You may want to implement the ManyToOne stuff with a Ride entity first, along with GET and POST so that you can see what the returned JSON looks like.

If I get some time, I'll see if I can put a spike together to illustrate the ideas.